streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java

```diff
@@ -477,6 +476,9 @@ public K peekNextKey() {
             super(storeName, iter, serdes);
             iter.seek(serdes.rawKey(from));
             this.rawToKey = serdes.rawKey(to);
+            if (this.rawToKey == null) {
+                throw new NullPointerException("RocksDBRangeIterator: ToKey is null");
```

Comment:
Better include the original `to` as well in the exception message in case the serde had a bug that serialize a non-null key to null.